frontend/tests/event.js

```diff
@@ -76,10 +76,14 @@ Quand("je découpe la glose en passages numérotés et que je me focalise sur la
 
 Quand("je remplace le contenu de la glose par ce qui suit et que je me focalise sur la glose :", (markdown) => {
   cy.click_on_text('content', '<TEXT>');
-  cy.get('textarea').type('{selectAll} ' + markdown.replaceAll(/[{}]/g, (x)=>`{${x}}`)).blur();
+  cy.get('textarea').type('{selectAll} ' + markdown.replaceAll(/[{}]/g, (x) => `{${x}}`)).blur();
```

Comment:
While pertinent, this coding style fix is not related with the current issue and hence should not be included in this branch.